packages/manager/modules/dbaas-logs/src/logs/translations/Messages_fr_FR.json

```diff
@@ -51,16 +51,49 @@
+  "logs_welcome_title": "Logs Data Platform",
+  "logs_welcome_description_title_1": "Une solution complète, sécurisée et à la demande pour gérer vos logs",
+  "logs_welcome_description_1": "Augmentez la visibilité au sein de vos environnements applicatifs, en collectant, traitant, analysant et stockant vos logs dans une plateforme complète et managée. L'analyse de ces journaux est essentielle afin de maintenir le bon fonctionnement de votre infrastructure et de vos applications. Nous vous proposons donc un service clefs en main, économique et évolutif, qui vous permet de vous concentrer sur votre cœur de métier.",
+  "logs_welcome_description_title_2": "Tarification claire",
+  "logs_welcome_description_2": "Bénéficiez d'une solution de gestion des logs complète et à la demande. Pour une utilisation avancée, il vous suffit d’ajouter des options à votre plateforme. Et grâce à des prix prévisibles ainsi qu’à un trafic illimité, vous ne serez jamais surpris par votre facture mensuelle. D’ailleurs, vous profitez de seuils et d’alertes pour optimiser le suivi de votre consommation.",
   "logs_welcome_try": "Commander",
+  "logs_welcome_guides_overview_title": "Présentation de Logs Data Platform",
+  "logs_welcome_guides_quick_start_title": "Utiliser Logs Data Platform",
+  "logs_welcome_guides_community_title": "Échanger sur Logs Data Platform",
   "logs_welcome_graylog_webui": "Interface web Graylog",
   "logs_welcome_grafana_saas": "Grafana SaaS",
   "logs_welcome_kibana_supported": "Compatible Kibana",
   "logs_welcome_product_page": "Page produit",
   "logs_welcome_documentation": "Documentation",
   "logs_welcome_forum": "Forum",
+  "logs_order_title": "Création de compte Logs Data Platform",
+  "logs_order_subtitle": "Créez votre compte Logs Data Platform pour accéder à toutes les fonctionnalités",
+  "logs_order_selection_title": "Choisissez l’option Logs Data Platform qui correspond à vos besoins",
+  "logs_order_selection_subtitle": "Gérez, sécurisez et ne payez que ce que vous utilisez",
+  "logs_order_selection_details_standard_title": "Standard",
+  "logs_order_selection_details_standard_description": "Inclus gratuitement et de façon illimitée :",
+  "logs_order_selection_details_standard_description_1": "tableau de bord et flux Graylog",
+  "logs_order_selection_details_standard_description_2": "contrôle d’accès à base de rôles",
+  "logs_order_selection_details_standard_description_3": "alias Elasticsearch",
+  "logs_order_selection_details_standard_description_4": "appels API Graylog et Elasticsearch",
+  "logs_order_selection_details_standard_description_5": "alertes",
+  "logs_order_selection_details_standard_description_6": "suivi des logs en temps réel.",
+  "logs_order_selection_details_standard_price": "Les 100 premiers Go : {{price}} HT/Go/mois",
```

Comment:
Are we sure that prices will always be monthly based ?

Reply:
on spec; offer is free and pay as you go, alternative is enterprise edition with 200€.  you think spec can be evolve on this part ? if we have a base price, we need to add a line because spec have not base base displayed for standard.